public/documents/fari-rpgs/charge-rpg.md

```diff
@@ -2334,6 +2334,300 @@ The original idea comes from [Ironsworn](https://ironswornrpg.com)'s vows. `Goal
+
+**Positive (+) Supporting Details**
+
+Positive `supporting details` for allies could be people or groups that are both friendly to the party and have the means to be helpful. 
+
+**Neutral (0) Supporting Details**
+
+Neutral `supporting details` for allies could be people that are willing to help but don’t have the means or maybe a group that hasn’t decided for sure whether the party is worthy of being helped. 
+
+**Negative (-) Supporting Details**
+
+Negative `supporting details` for allies could be people that you’re connected to that cause more trouble for the party than they might be worth, or estranged relationships. 
+
+#### 4. Adversaries
+
+This represents people or groups that are against the party, or possibly even harsh conditions & impending doom. While typically negative sounding, positive leaning `dispositions` could represent friendly competition or forces of nature that can be treated as both friend and foe. A positive `key concept` might be something like “with friends like these, who needs enemies?”, while a negative one might be something like “hunted by nearly everyone”. 
+
+**Positive (+) Supporting Details**
+
+Positive `supporting details` for adversaries could be people or groups that might still help the party if they were in mortal danger, sworn enemies that are comically bad at their jobs, or maybe enemies that don’t yet realize the party exists.
+
+**Neutral (0) Supporting Details**
+
+Neutral `supporting details` for adversaries could be enemies that have bigger fish to fry and consider the party beneath them, or powerful desert sandstorms that can be used to the party’s advantage if they can survive. 
+
+**Negative (-) Supporting Details**
+
+Negative `supporting details` for adversaries might include powerful enemies sworn to destroy the party, or impending doom of an inbound asteroid.  
+
+## Danger and Progress
+
+> #### Designer Notes
+> 
+> The pace at which the `supporting details` of the party change can be a dial for changing the feel of the game, especially in games where additional mechanical benefits are offered for building up positive `supporting details`.  By default, the intended pace is an average of no more than one detail changing per session.
+
+The `party sheet` holds two clocks which fight against each other, one dragging the party down, and the other pulling the party up. By default, both of these clocks have eight segments. 
+
+### Taking Consequences
+
+The GM or players may suggest that marking a consequence on the `party sheet` makes sense. Generally this should represent a major consequence as it can have long-term implications, and should not be taken lightly - a couple of times per session is probably a good rule of thumb. This adds segments to the danger clock of the party sheet.
```

Comment:
> The GM or players may suggest that marking a consequence on the `party sheet` makes sense
OK, when should this happen? I have the impression the information in this paragraphe should be switched up like.

Like:
- There will be times when something very bad happens to this party
- When this happens, the group can agree to tick a danger clock on the party sheet (I see you are talking about "marking a consequence" but I don't think its a thing??)
- This should usually not happen more than once per session.